### ISSUE TITLE
Modal Overlay Issue

### ISSUE THREAD

**Issue by sagar-go** [grommet/grommet#6769]
### Expected Behavior

When there are two Modals, they are appearing on top of each other. Is there a way, we can have two separate Modals at same time? 

Requirement : When we open secondary Modal, the overlay should remain same. Current implementation is that, primary Modal have default overlay of rgba(0,0,0,0.5), when we open secondary Modal, there is another overlay of rgba(0,0,0,0.5) but on behind the primary modal's overlay.

When we open primary Modal, overlay of 0.5 gets applied on transparent/white background, but overlay of secondary modal gets applied on primary modal's overlay.

### Actual Behavior

Both Modal should have same overlay. Is there a way, we can achieve it?

### URL, screen shot, or Codepen exhibiting the issue

**https://storybook.grommet.io/?path=/story/layout-layer-center--center-layer**

Please notice the overlay background of primary(1st) and sendondary(2nd) Modal. Second one is bit darker, compared to first one. 

I have tried Theme.overlay.background, but it is changing for both the Modal, not just for secondary Modal.

**Comment by jcfilben** [comment]
Het @sagar-go you could set the `theme.layer.overlay.background` to a non-transparent color, this way you will have a consistent background and won't run into the issue of the two transparent backgrounds causing a darker background the more layers you have open.

**Comment by sagar-go** [comment]
I tried that it is not working, because as I mentioned, if you change background of one Modal it will affect other Modal's background as well, since they both are same Modal, but just on top of each other.

**Comment by sagar-go** [comment]
The structure that currently we have is :

1 - Body Background white 
2 - Primary Modal Overlay (rgba 0.3)
3 - Primary Modal
4: Secondary Modal overlay (rgba(0.3)
5: Secondary Modal

Now if I make the secondary Modal overlay to transparent, then Primary Modal's overlay also get transparent and hence we are left with Body background white. 

Note : Modal's are on top of each other, they are not two separate Modal's. Changing a Modal's background affect Both Modal.

**Comment by jcfilben** [comment]
> @jcfilben I tried that it is not working, because as I mentioned, if you change background of one Modal it will affect other Modal's background as well, since they both are same Modal, but just on top of each other.

This is true for transparent backgrounds but shouldn't be true for solid colored backgrounds. If you want to use a transparent color a possible work around could be to wrap the second layer with ThemeContext and set the `theme.layer.overlay.background` to be completely transparent.

**Comment by jcfilben** [comment]
Hey @sagar-go just wanted to check in and see if this solution worked for you. If so I think we can go ahead and close the issue.

**Comment by sagar-go** [comment]
It did work for transparent color(which is our requirement) but it is
working for solids.
